Amplify/DevMenu/View/EnvironmentInfoRow.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+/// View corresponding to each row in Environment Information screen
+@available(iOS 13.0.0, *)
+struct EnvironmentInfoRow: View {
+    var rowItem: EnvironmentInfoItem
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(rowItem.key).bold()
+            Text(rowItem.value)
+        }
+    }
+}
```

Comment:
this view could be more generic, so both "device" and "environment" info could use it